tox.ini

```diff
@@ -38,3 +38,11 @@ skip_install = true
 deps = isort
 commands = isort -c -rc -df spid-testenv.py testenv
 skip_install = true
+
+[flake8]
+# Flake8 can be configured here.
+ignore =
+  E501  # Line too long
+  W503  # line break before binary operator
```

Comment:
Probabilmente nel caso di W503 vale la pena correggere le occorrenze dell'errore nel codice, anziché metterlo nell'ignore list.

Reply:
D'accordo. 

Non essendo in pianta stabile nel team di sviluppo ho usato il commento per permettervi di decidere una linea. In base alle vostre priorità potete:

  - accettare ed aprire un nuovo ticket
  - fixare e notificarmi in modo che io modifichi la PR.

Reply:
Secondo me conviene fixare tutte le occorrenze di W503 (se sono molto può aver senso fixarle a parte, come preferisci)